templates/init/functions/javascript/package.nolint.json

```diff
@@ -9,14 +9,14 @@
     "logs": "firebase functions:log"
   },
   "engines": {
-    "node": "8"
+    "node": "10"
   },
   "dependencies": {
-    "firebase-admin": "^8.9.0",
-    "firebase-functions": "^3.3.0"
+    "firebase-admin": "^8.10.0",
+    "firebase-functions": "^3.5.0"
```

Comment:
Should this be https://github.com/firebase/firebase-functions/releases/tag/v3.6.0